src/components/Info.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import info from './icons/info.svg';
+import './Info.css';
+
+export default class Info extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false
+    };
+    this.toggleInfo = this.toggleInfo.bind(this);
+  }
+
+  toggleInfo() {
+    this.setState({
+      show: !this.state.show
+    });
+  }
+
+  render() {
+    const displayClass = this.state.show ? 'Show' : 'Hide';
+    return (
+      <div className="Info-container">
+        <div className="Info-header">
+          <h2>{this.props.title}</h2>
```

Comment:
Not a fan of having your 'Info' component also contain your header, as it's not intuitive that those two be intrinsically tied. What if we want to use an info icon not associated with an h2 header? It's brittle.

I would decouple the h2 from this component and rename it `InfoIcon` for clarity, still passing in the "blurb" (infoBody?).

Reply:
Done